README.rst

```diff
@@ -1,21 +1,81 @@
-TunableLaser
-============
+###############
```

Comment:
The thinking here is that this will be a prominent first piece of documentation that developer/user will see. Although I do agree with you about the duplication of information. Should I remove the use case duplicated information?

Reply:
Yes, but also link the documentation so they can go find usage information etc.
I think we need to standardize what is to go in the README files... but that's not for this ticket.

Reply:
Yeah, I'm trying to see what sticks here. Added a link to the website.